doc/decent/architectures.md

```diff
@@ -33,9 +35,13 @@ for {
 }
 ```
 
-All of the above steps are relatively simple when using Noms DB with the  possible exception of "merge", but in most cases, that is simple, too. If the data being managed is modeled as a CRDT, then merging can be done automatically. In the case where conflicts can exist, Noms can resolve them in favor of one client or the other, ir if that's not sufficient, they can be resolved by application specific code.
+Noms has a default merge policy that covers many classes of concurrent operations. If the application restricts itself to only operations that are mergeable by this policy, then Noms can automatically merge all concurrent changes. In this case, the entire database is effectively a CRDT.
```

Comment:
Link "merge policy" to the code comments in three_way.go that describe the merge policy.